packages/vite/src/node/optimizer/scan.ts

```diff
@@ -358,7 +358,8 @@ export const commentRE = /<!--.*?-->/gs
 const srcRE = /\bsrc\s*=\s*(?:"([^"]+)"|'([^']+)'|([^\s'">]+))/i
 const typeRE = /\btype\s*=\s*(?:"([^"]+)"|'([^']+)'|([^\s'">]+))/i
 const langRE = /\blang\s*=\s*(?:"([^"]+)"|'([^']+)'|([^\s'">]+))/i
-const contextRE = /\bcontext\s*=\s*(?:"([^"]+)"|'([^']+)'|([^\s'">]+))/i
+const svelteScriptModuleRE =
```

Comment:
i'd prefer if this regex was split in 2 and we keep the previous style, adding a new regex for module attribute. 

The new one is a lot more complex and its hard to understand what exactly it does compared to the other two. If the script tag is  not properly closed and the content gets long, the performance can also suffer.

not for this PR, but a future change to scan, the regexes could be replaced by a single pass attribute parser.